src/plugins/dataflow-tool/nodes/controls/dropdown-list-control.tsx

```diff
@@ -75,19 +76,19 @@ export class DropdownListControl extends Rete.Control {
                                     });
       const option = options.find((opt) => optionValue(opt) === val);
       const name = option ? option.name : val;
-      const icon = option && option.icon ? `#${option.icon}` : null;
+      const icon = option && option.icon ? option.icon({}) : null;
```

Comment:
Can be simplified to something like:
```suggestion
      const name = option?.name ?? val;
      const icon = option?.icon?.({}) || null;
```